### PR TITLE
Add GitHub actions

### PR DESCRIPTION
**Pull request by grisu48**
Add GitHub actions and remove Travis

### CODE REVIEW

File: .github/workflows/pcw.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+name: public cloud watcher
+'on': push
+
+jobs:
+  pcw:
+    name: public cloud watcher
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
```
Comment (asmorodskyi) [inline .github/workflows/pcw.yaml:12]:
```suggestion
        python-version: [3.6, 3.8]
```

Comment (grisu48) [reply .github/workflows/pcw.yaml]: :+1:

**Comment by asmorodskyi** [comment]
awesome :+1:  thanks for this never could reach this ...

File: .github/workflows/pcw.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+
+name: public cloud watcher
+'on': push
+
+jobs:
+  pcw:
+    name: public cloud watcher
+    runs-on: ubuntu-latest
```
[the remark this answers is not in the document]
Comment (grisu48) [reply .github/workflows/pcw.yaml:9]: Not sure if those are even supported. I'm following the official instructions here: https://docs.github.com/en/actions/guides/building-and-testing-python

But once it runs we can try it out ;-)

File: .github/workflows/pcw.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+
+name: public cloud watcher
+'on': push
```
Comment (ggkioulis) [inline .github/workflows/pcw.yaml:4]: should this be `'on'` instead of `on`?

Comment (grisu48) [reply .github/workflows/pcw.yaml:4]: Both is supported, just with `'on'` yamllint doesn't complain about a boolean statement should be `true` or `false`. But I'll have another check!

Comment (grisu48) [reply .github/workflows/pcw.yaml]: This one:

    .github/workflows/pcw.yaml
      4:1       warning  truthy value should be one of [false, true]  (truthy)

Comment (grisu48) [reply .github/workflows/pcw.yaml]: Still doesn't want to :disappointed:

File: .github/workflows/pcw.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (asmorodskyi) [reply .github/workflows/pcw.yaml:9]: ok let's follow the flow , we can improve this later ...

**Review by asmorodskyi — APPROVED** [review]
LGTM , thanks a lot :+1:

**Comment by grisu48** [comment]
Don't know why the Actions are not scheduled in this PR, but they are scheduled in the [branch](https://github.com/grisu48/pcw/actions/runs/704860628)

**Pull request merged**